### DISCUSSION TITLE
Fast and Accurate Image Super Resolution by Deep CNN with Skip Connection and Network in Network

### DISCUSSION THREAD

**Discussion by harshraj22** [harshraj22/DeepLearning#14] [Paper Review]
# DCSCN [:notebook: ](https://arxiv.org/abs/1707.05425)
<ins> CVPR 2017 </ins>

## Architecture Details:
- They proposed CNN model focusses on learning the residuals between the bicubic interpolation of the LR image and the HR original image.

- The architecture consists of standard convolutional blocks with skip connections. The authors argue, skip connections help in stabilising the model, while passing on the information extracted from earlier layers to the later layers.
![image](https://user-images.githubusercontent.com/46635452/126643945-4823c630-c7a2-4316-a51d-d44c56bdc546.png)

- 1 x 1 conv layer is introduced at the end in order to reduce feature maps size without increasing the model complexity.
Mean Squared Error was used as a loss function along with L2 regularization.

## Dataset Details:
- The model was trained on a mix of [DIV2K](https://data.vision.ee.ethz.ch/cvl/DIV2K/) dataset (91 images out of 800 images) and [BSD](https://www2.eecs.berkeley.edu/Research/Projects/CS/vision/bsds/) dataset (200 images). [SET5](https://paperswithcode.com/dataset/set5) dataset was used for validation.
- They performed data augmentations like flipping horizontally, vertically and both horizontally and vertically to make 3 more images for each image.
- Thus the total size of training dataset was 1,164 images.
